domain/robstoll-lib/atrium-domain-robstoll-lib-jvm/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/pathAssertions.kt

```diff
@@ -27,3 +28,9 @@ fun <T : Path> _parent(assertionContainer: Expect<T>): ExtractedFeaturePostStep<
         .withCheck { it.parent != null }
         .withFeatureExtraction { it.parent }
         .build()
+
+fun <T : Path> _fileNameWithoutExtension(assertionContainer: Expect<T>): ExtractedFeaturePostStep<T, String> =
+    ExpectImpl.feature.manualFeature(
+        assertionContainer,
+        DescriptionPathAssertion.FILE_NAME_WITHOUT_EXTENSION
+    ) { fileNameWithoutExtension }
```

Comment:
This is IntelliJ default formatter, is it ok?

Reply:
I would write (requires an additional import for FILE_NAME_WITHOUT_EXTENSION)
```
fun <T : Path> _fileNameWithoutExtension(assertionContainer: Expect<T>): ExtractedFeaturePostStep<T, String> =
    ExpectImpl.feature.manualFeature(assertionContainer, FILE_NAME_WITHOUT_EXTENSION) { fileNameWithoutExtension }
```